apollo-client/src/main/java/com/ctrip/framework/apollo/model/ConfigChangeEvent.java

```diff
@@ -55,4 +55,8 @@ public boolean isChanged(String key) {
   public String getNamespace() {
     return m_namespace;
   }
+
+  public Map<String, ConfigChange> getChangeValue(){
```

Comment:
I think this is not necessary, as we could use `com.ctrip.framework.apollo.model.ConfigChangeEvent#getChange`

Reply:
sure. ok